### ISSUE TITLE
why does my package have so many downloads?

### ISSUE THREAD

**Issue by absurdmoe** [npm/www#270]
I just created a package, https://www.npmjs.com/package/shoebilljs, less than 48 hours ago and it already has 580 downloads. I haven't shared this package to a lot of people, so that number seems inaccurate. Can anyone tell me how to view the actual number of unique users or downloads

**Comment by andreasonny83** [comment]
yes, I was questioning the same. It seems almost impossible to have so many downloads for something nobody knows its existence yet. Also, my GitHub repository for that npm package has still 0 stars. I still have no answers to that question, unfortunately.

**Comment by soldair** [comment]
The downloads are probably from replicators/mirrors who just download everything every time there is a change.

 We don't break down uniques or counts by version but I'm sure we'll do more in this space eventually.
